langwatch/notebooks/scenario_end_to_end.ipynb

```
@@ -0,0 +1,314 @@
+    "\n",
+    "resp = S.post(f\"{BASE_URL}/api/prompts\", json={\n",
+    "    \"handle\": f\"acmebank-support-{int(time.time())}\",\n",
+    "    \"scope\": \"PROJECT\",\n",
+    "    \"prompt\": SYSTEM_PROMPT,\n",
+    "    \"messages\": [{\"role\": \"user\", \"content\": \"{{input}}\"}],\n",
+    "    \"inputs\": [{\"identifier\": \"input\", \"type\": \"str\"}],\n",
+    "    \"outputs\": [{\"identifier\": \"output\", \"type\": \"str\"}],\n",
+    "    \"model\": os.environ.get(\"LANGWATCH_MODEL\", \"openai/gpt-5-mini\"),\n",
+    "})\n",
+    "resp.raise_for_status()\n",
+    "prompt_id = resp.json()[\"id\"]\n",
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)